src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -312,7 +310,7 @@ export class EventManager implements IEventManager {
 
     if (contextId === null) {
       // For global subscriptions, events buffered in each context should be sent back.
-      Array.from(this.#eventToContextsMap.get(eventName)?.keys() ?? [])
+      Array.from(this.#eventToContextsMap.get(eventName).keys() ?? [])
```

Comment:
This will create a set for each event user subscribed to, not necessary existing. But I guess it's fine.

Reply:
Yes, in the end the loop would just be skipped as the set is empty. Unless we rely on the empty set semantics elsewhere, it shouldn't be an issue.